packages/launchpad/src/setup/EnvironmentSetup.vue

```diff
@@ -1,12 +1,52 @@
 <template>
+  <Alert
```

Comment:
nitpick: Personally I would have pulled this out as a standalone component to improve testability, but not required to do so